src/index.js

```diff
@@ -473,6 +496,256 @@ const initialize = async () => {
     }
   }
 
+  /**
+   * Ethereum Signature
+   */
+  ethSign.onclick = async () => {
+    try {
+      // const msg = 'Sample message to hash for signature'
+      // const msgHash = keccak256(msg)
+      const msg = '0x879a053d4800c6354e76c7985a865d2922c82fb5b3f4577b2fe08b998954f2e0'
+      const ethResult = await ethereum.request({
+        method: 'eth_sign',
+        params: [accounts[0], msg],
+      })
+      console.log('signed:', JSON.stringify(ethResult))
+      ethSignResult.innerHTML = JSON.stringify(ethResult)
+    } catch (err) {
+      console.log(err)
+    }
+  }
+
+  /**
+     * Personal Sign
+     */
+
+  personalSignTest.onclick = async () => {
+    const exampleMessage = 'Sample message for personal sign test'
+    try {
+      const from = accounts[0]
+      const msg = `0x${Buffer.from(exampleMessage, 'utf8').toString('hex')}`
+      const result = await ethereum.request({
+        method: 'personal_sign',
+        params: [msg, from, 'Example password'],
+      })
+      console.log('recovering!!!')
+      const recoveredAddr = recoverPersonalSignature({
+        'data': msg,
+        'sig': result,
+      })
+      if (recoveredAddr === from) {
+        console.log(`SigUtil Successfully verified signer as ${recoveredAddr}`)
+      } else {
+        console.log(`SigUtil Failed to verify signer when comparing ${recoveredAddr} to ${from}`)
+        console.log(`Failed comparing ${recoveredAddr} to ${from}`)
+      }
+    } catch (err) {
+      console.error(err)
+    }
+  }
+
+  /**
+   * Personal ecRecover Test
+   */
+  personalRecoverTest.onclick = async () => {
+    const exampleMessage = 'Sample message for personal ec Recover Test'
```

Comment:
This should use the existing `personalSign` example message:
```suggestion
      const exampleMessage = 'Example `personal_sign` message'
```